src/pages/coaches/eva-hoffmann.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Eva Hoffmann
+title: Software Engineer
+companyName: DAZN
+companyLink: https://engineering.dazn.com/
+gitHub: https://github.com/evandyou
+medium: ''
+twitter: https://twitter.com/evandyou
+linkedIn: https://uk.linkedin.com/in/evamhoffmann
+blockquote:
+  Everyone can learn how to program, and I enjoy supporting others to learn and expand their skill set. Programming is
+  such an important skill set and I am particularly passionate
+  about diversity in tech.
+imageDescription: React GraphQL Academy coach Eva Hoffmann, smiling.
+imageSrc: https://firebasestorage.googleapis.com/v0/b/reactjsacademy-react.appspot.com/o/team%2Feva.jpg?alt=media
+tags: []
+---
+
+<ul>
```

Comment:
@ric9176 why don't we use markdown for the list? https://www.markdownguide.org/basic-syntax/#unordered-lists